src/modules/ui/components/floating_menu/svg_package.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable */
```

Comment:
It's just a bundle with svg images which I use in my menu. I have two reasons to put them into one file:

- to don't bloat src with a bunch of image files

- to don't add new dependency to project (I need here `file-loader` or `base64-image-loader`)

So I just prebuild it manually with `base64-image-loader`. Maybe it could be usefull later if you'll want to extend it with more images? But if you don't like such approach could you suggest the better way how to get this images?